src/plugins/workspace/public/components/workspace_overview/all_get_started_cards.ts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DEFAULT_APP_CATEGORIES } from '../../../../../core/public';
+import { GetStartCard } from './types';
+
+/**
+ * All getting start cards
+ */
+export const getStartCards: GetStartCard[] = [
+  // getStarted
+  {
+    appId: 'home',
+    featureDescription: 'Discover pre-loaded datasets before adding your own.',
```

Comment:
Seems the name and description is not wrapped by i18n. If it is a workaround for 2.14 then forget it.